posthog/api/test/test_signup.py

```diff
@@ -0,0 +1,772 @@
+import datetime
```

Comment:
Test cases kept equal from `test_organization` unless otherwise noted within the file.